kapitan/resources.py

```diff
@@ -304,6 +306,52 @@ def generate_inventory(args):
         sys.exit(1)
 
 
+def get_inventory(inventory_path, ignore_class_notfound=False, targets=[]):
+    """
+    generic inventory function that makes inventory backend pluggable
+    default backend is reclass
+    """
+
+    # if inventory is already cached theres nothing to do
+    if cached.inv:
+        return cached.inv
+
+    # get parsed args from cached.py
+    args = list(cached.args.values())[0]
+    use_omegaconf = args.omegaconf
+    try:
+        migrate_omegaconf = args.migrate
```

Comment:
not really use why we need to do this inside a try?

Reply:
use perhaps:

```
migrate_omegaconf = args.get("migrate", False)
```